src/XRP/issued-currency-client.ts

```diff
@@ -531,6 +531,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Re-enables rippling on the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
```

Comment:
again, is this true??  I don't think it is

Reply:
it's not, removed